cli/script/command-executor.ts

```diff
@@ -1169,21 +1113,15 @@ export var release = (command: cli.IReleaseCommand): Promise<void> => {
         isMandatory: command.mandatory,
         rollout: command.rollout
     };
-
-    return getPackageFilePromise
-        .then((file: IPackageFile): Promise<void> => {
+    
+    return Promise<void>((resolve, reject) => {
```

Comment:
I don't think this promise will ever finish, because `resolve`/`reject` is never called. I think if you just remove this enclosing promise (leaving `return sdk.isAuthenticated()...` below), it'll work perfectly though!